src/app/containers/StakePage/components/HistoryEventsTable.tsx

```diff
@@ -222,35 +153,42 @@ interface HisoryAsset {
   index: number;
 }
 
-const HisoryTableAsset: React.FC<HisoryAsset> = ({ item, index }) => {
+const HisoryTableAsset: React.FC<HisoryAsset> = ({ item }) => {
```

Comment:
You have a typo here, it should be `HistoryTableAsset` and `HistoryAsset`.

Reply:
yeah, fixed